src/Areas/Extension/Options/AzqrOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using AzureMcp.Options;
+
+namespace AzureMcp.Areas.Extension.Options;
+
+public class AzqrOptions : GlobalOptions
+{
+    [JsonPropertyName(ExtensionOptionDefinitions.Azqr.SubscriptionIdName)]
+    public string SubscriptionId { get; set; } = string.Empty;
+
+    [JsonPropertyName(ExtensionOptionDefinitions.Azqr.ResourceGroupIdName)]
+    public string? ResourceGroupName { get; set; }
```

Comment:
You won't need these once you use Subscription and ResourceGroup as they are already in other classes in the tree

Reply:
Removed.